int32 sayi1
int32 sayi2
---
int32 sonuc